src/test/run-cli-tests

```diff
@@ -1,6 +1,27 @@
 #!/bin/sh
 set -e
 
+function run_test() {
```

Comment:
i think we need to either drop the bashism or use bash instead.

Reply:
As we do have already functions in this file, ack to switch to bash ?

Reply:
@tchaikov Sorry I mixed with the other bash aware scripts. Fixed.